exercises/practice/eliuds-eggs/.meta/reference/eliuds_eggs.bal

```diff
@@ -0,0 +1,13 @@
+public function eggCount(int displayValue) returns int {
+    int eggs = 0;
+    int current = displayValue;
+    while current != 0 {
+        if current % 2 == 1 {
+            eggs += 1;
+        }
+            
+        current /= 2;
+    }
```

Comment:
Maybe we can use following:
```suggestion
    while current != 0 {
        eggs = current % 2 == 1 ? eggs + 1 : eggs;  
        current /= 2;
    }
```

Reply:
I have a weak preference for the former since I'm not sure the ternary makes the code clearer. Students might look at this example to compare to theirs after passing so I'd favor readability.

Reply:
Make sense.